scripts/data.json

```diff
@@ -642,6 +649,13 @@
     }
   ],
   "U": [
+    {
+      "name": "UiKit",
+      "url": "https://getuikit.com/",
+      "type": "CSS Framework",
+      "year_created": 2007,
+      "year_created_source": "https://bluewhaleapps.com/blog/uikit-vs-swiftui#:~:text=UIKit%20is%20the%20more%20mature,apps%20for%20the%20IOS%20platform."
```

Comment:
Please try to use official sources like npm package, github release or official blogpost. 

Remove unnecessary anchor for text highlight.

Reply:
Alright... But I used that url because it showed the date it was created. 

I will look into it or possibly delete it.